requirements.txt

```diff
@@ -1,2 +1,4 @@
 numpy 
-numpy-quaternion==2020.5.11.13.33.35
\ No newline at end of file
+numpy-quaternion==2020.5.11.13.33.35
+scipy
+numba
```

Comment:
These packages are not required for building or running the simulator, and do not add any improvement.